reflex/utils/pyi_generator.py

```diff
@@ -159,13 +159,15 @@ def _get_type_hint(value, type_hint_globals, is_optional=True) -> str:
             ]
         )
 
-        if value.__name__ not in type_hint_globals:
+        if value.__module__ == "builtins":
+            res = f"{value.__name__}[{', '.join(inner_container_type_args)}]"
```

Comment:
Can we not duplicate this line, it's the same as the else case.

Reply:
fixed this